### PR TITLE
Add IT with security for alerting

### PR DESCRIPTION
**Pull request by gaiksaya**
*Issue #, if available:* https://github.com/opendistro-for-elasticsearch/opendistro-build/issues/346

*Description of changes:* Hook alerting IT with security to all distributions

*Test Results:*

- DEB: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/995559159?check_suite_focus=true
- RPM: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/212802592
- TAR: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/996047260?check_suite_focus=true
- DOCKER: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/996085569?check_suite_focus=true
- WINDOWS: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/996150073?check_suite_focus=true

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-build-rpm.yml [hunk 1]
```diff
@@ -334,7 +334,29 @@ jobs:
           .github/scripts/setup_runners_service.sh rpm --es
           export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../anomaly-detection; pwd
           ./gradlew :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin
-
+  
+  Test-ALERTING:
```
Comment (peterzhuamazon) [inline .github/workflows/staging-build-rpm.yml:338]: Hello. Please put the new self host runner in provision and cleanup, and also add the needs of this into cleanup.

Comment (weicongs-amazon) [reply .github/workflows/staging-build-rpm.yml:338]: good point.  @gaiksaya, it seems that you just forgot to add them. I see the logic is already in the testing workflow file.

Comment (gaiksaya) [reply .github/workflows/staging-build-rpm.yml:338]: Aah! My bad let me add

Comment (gaiksaya) [reply .github/workflows/staging-build-rpm.yml:338]: Added!

**Comment by skkosuri-amzn** [comment]
Thanks for adding this. 👍 @gaiksaya

**Pull request merged**